Userland/Libraries/LibWeb/DOM/DocumentLoading.cpp

```diff
@@ -78,6 +78,7 @@ static WebIDL::ExceptionOr<JS::NonnullGCPtr<DOM::Document>> load_markdown_docume
                 return;
 
             auto parser = HTML::HTMLParser::create(document, markdown_document->render_to_html(extra_head_contents), "utf-8");
+            document.set_url(url);
             parser->run(document.url());
```

Comment:
Seems like we should just do:
```c++
auto parser = HTML::HTMLParser::create(document, markdown_document->render_to_html(extra_head_contents), "utf-8");
parser->run(url);
```

Because calling `document.set_url` is the first thing and only thing that `run` does with the URL.